source/test/E2E.test.js

```diff
@@ -0,0 +1,98 @@
+
+describe('Basic web functionality check', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8436700)
```suggestion
describe("Basic web functionality check", () => {
```